src/Star.cpp

```diff
@@ -1,14 +1,17 @@
 // Copyright © 2008-2019 Pioneer Developers. See AUTHORS.txt for details
 // Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
 
+#include <memory>
 #include "Star.h"
 
+#include "Frame.h"
 #include "Pi.h"
 #include "galaxy/StarSystem.h"
 #include "graphics/Graphics.h"
 #include "graphics/Renderer.h"
 #include "graphics/RenderState.h"
 #include "graphics/VertexArray.h"
+#include "graphics/VertexBuffer.h"
```

Comment:
Same as above

Reply:
Cleaned up these headers now. The frame one was a leftover from some debug rendering I was testing.